### PR TITLE
fix: track ETH and CAKE pair liquidity

### PR DESCRIPTION
**Pull request by honeyswapbsc**
this will fix certain pairs like BETH https://github.com/pancakeswap/pancake-swap-interface/pull/167 or renBTC

( Right now, if you want to trade BETH with BNB, it will show less liquidity on the exchange.pancakeswap.finance )

### CODE REVIEW

**Comment by honeyswapbsc** [comment]
Also adding CAKE pair as many projects and more than 4 PancakeSwap LP farm has CAKE pair https://pancakeswap.finance/farms.

This will show the CAKE LP liquidity by default, without adding them with import pool.

**Comment by honeyswapbsc** [comment]
Routing for specific pairs are now working well

Before: https://exchange.pancakeswap.finance/

before

After: https://pancakefix.honeyswap.fi/#/swap

after

**Comment by honeyswapbsc** [comment]
@hachiojidev @Chef-Chungus @ChefKai Could you review this? Thanks.

File: src/constants/index.ts [hunk 1]
```diff
@@ -10,13 +10,10 @@ type ChainTokenList = {
...
 export const USDT = new Token(ChainId.MAINNET, '0x55d398326f99059ff775485246999027b3197955', 18, 'USDT', 'Tether USD')
-export const UST = new Token(
-  ChainId.MAINNET,
-  '0x23396cf899ca06c4472205fc903bdb4de249d6fc',
-  18,
-  'UST',
-  'Wrapped UST Token'
-)
+export const UST = new Token(ChainId.MAINNET, '0x23396cf899ca06c4472205fc903bdb4de249d6fc', 18, 'UST', 'Wrapped UST Token')
+export const BTCB = new Token(ChainId.MAINNET, '0x7130d2a12b9bcbfae4f2634d864a1ee1ce3ead9c', 18, 'BTCB', 'BTCB Token')
```
Comment (Chef-Chungus) [inline src/constants/index.ts:14]: We don't need BTCB

File: src/constants/index.ts [hunk 2]
```diff
@@ -26,7 +23,7 @@ const WETH_ONLY: ChainTokenList = {
 // used to construct intermediary pairs for trading
 export const BASES_TO_CHECK_TRADES_AGAINST: ChainTokenList = {
   ...WETH_ONLY,
-  [ChainId.MAINNET]: [...WETH_ONLY[ChainId.MAINNET], DAI, BUSD, USDT, UST],
+  [ChainId.MAINNET]: [...WETH_ONLY[ChainId.MAINNET], DAI, BUSD, USDT, UST, BTCB, ETH, CAKE],
```
Comment (Chef-Chungus) [inline src/constants/index.ts:26]: Same here

File: src/constants/index.ts [hunk 3]
```diff
@@ -46,7 +43,7 @@ export const SUGGESTED_BASES: ChainTokenList = {
 // used to construct the list of all pairs we consider by default in the frontend
 export const BASES_TO_TRACK_LIQUIDITY_FOR: ChainTokenList = {
   ...WETH_ONLY,
-  [ChainId.MAINNET]: [...WETH_ONLY[ChainId.MAINNET], DAI, BUSD, USDT],
+  [ChainId.MAINNET]: [...WETH_ONLY[ChainId.MAINNET], DAI, BUSD, USDT, UST, BTCB, ETH, CAKE],
```
Comment (Chef-Chungus) [inline src/constants/index.ts:46]: Same here

**Comment by honeyswapbsc** [comment]
@Chef-Chungus I have addressed your comments, please review

**Pull request closed without merging**